FifthLesson/VectorProgram/VectorProgram/Controller/VectorHelper.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using VectorProgram.Model;
+
+namespace VectorProgram.Controller
+{
+    public static class VectorHelper
+    {
+        public static double AngleBetweenVectors(Vector ferst, Vector second)
+        {
+            var scalarMultiple = ferst.FirstCoordinate * second.FirstCoordinate + ferst.SecondCoordinate * second.SecondCoordinate + ferst.ThirdCoordinate + second.ThirdCoordinate;
+
+            var moduleFirstVector = Math.Sqrt(Math.Pow(ferst.FirstCoordinate, 2) + Math.Pow(ferst.SecondCoordinate, 2) + Math.Pow(ferst.ThirdCoordinate, 2));
```

Comment:
I think it would be better if modules' vector will be calculated in private method

Reply:
I mean that the method **AngleBetweenVectors** should not know how the vectors' module is calculated.

Reply:
Corrected

Reply:
Added method **CalculateModuleVector**